### PR TITLE
Data: Hide relationships

### PR DESCRIPTION
**Pull request by gblakeman**
* Adds _Newsletter_ and _Event_ assignment types + positions the entity cards
* Hides “subscribers” on _Newsletter_ cards
* Hides “attendees” on _Event_ cards

----

Includes #129 (already reviewed)

### CODE REVIEW

**Review by lynnerang — APPROVED** [review]
LGTM!

File: src/Tools.ts [hunk 1]
```diff
@@ -138,13 +139,37 @@ const getNodeContentInfo = (node: SceneNode): RealishContentInfo => {
...
+      contentInfo.assignment = updateLegacyAssignment(contentInfo.assignment);
+    }
     return contentInfo as RealishContentInfo;
   }
 
-  // fallback for old string assignment
-  return { assignment: contentInfo };
+  // --- fallback for old string assignment
+  let flatAssignment: RealishAssignment = contentInfo;
+  if (legacies.flatMap((legacy: RealishLegacy) => legacy.old).includes(flatAssignment)) {
```
Comment (joeyyang) [inline src/Tools.ts:168]: can we break the result of this `.flatMap` condition into a separate line, e.g.:
```ts
const legaciesIncludesAssignments = legacies.flatMap(...);
if (legaciesIncludesAssignments) {
 ...
```

**Review by joeyyang — APPROVED** [review]
Readability nit but otherwise looks good!

File: src/Tools.ts (hunk 1 above)
Comment (gblakeman) [reply src/Tools.ts]: Yes, good call. It’s used twice too anyway, so worth pulling out separately.

**Pull request merged**